### PR TITLE
feat(permissions): Redirect on permission click

### PR DESCRIPTION
**Pull request by vdnj**
A new page opens when we click on a permission
with @Noemie21 
179173224-baac173d-e710-41f5-81dc-f34fedf82617

### CODE REVIEW

File: src/containers/Permission.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          <Typography variant="h1">
+            Application: {appName.toUpperCase()}
+          </Typography>
+          <Typography variant="h2">Permission: {permissionName}</Typography>
+          <List>
+            {queryResult.data[0].attributes.permissions[permissionName]
+              .verbs ? (
+              queryResult.data[0].attributes.permissions[
+                permissionName
+              ].verbs.map((el, i) => <ListItemText key={i}>{el}</ListItemText>)
```
Comment (trollepierre) [inline src/containers/Permission.jsx:44]: Let's try not to use index in key
```suggestion
              ].verbs.map((el) => <ListItemText key={el}>{el}</ListItemText>)
```

Comment (trollepierre) [reply src/containers/Permission.jsx:44]: Let's try to use full word (i don't know what is an "el")
```suggestion
              ].verbs.map((permission) => <ListItemText key={permission.name}>{permission}</ListItemText>)
```

File: src/containers/Permission.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      {isQueryLoading(queryResult) && !hasQueryBeenLoaded(queryResult) ? (
+        <Spinner size="large" className="u-flex u-flex-justify-center u-mt-1" />
+      ) : queryResult.fetchStatus === 'failed' ? (
+        <Typography variant="body1" className="u-mb-1-half">
+          {t('Permissions.failedRequest')}
+        </Typography>
+      ) : (
+        <div>
+          <Typography variant="h1">
+            Application: {appName.toUpperCase()}
```
Comment (trollepierre) [inline src/containers/Permission.jsx:36]: let's try to use translation everywhere
```suggestion
            {t('Permissions.Application')}{appName.toUpperCase()}
```

File: src/components/App.jsx [hunk 3]
```diff
@@ -45,9 +46,14 @@ export class App extends Component {
             <Route path="/storage" component={Storage} />
             <Route exact path="/permissions" component={Permissions} />
             <Route
+              exact
               path="/permissions/:app"
               component={PermissionsApplication}
             />
+            <Route
+              path="/permissions/:app/:permission"
+              component={Permission}
```
Comment (trollepierre) [inline src/components/App.jsx:55]: why not adding exact here?

Comment (vdnj) [reply src/components/App.jsx:55]: It is not needed in this case

File: src/containers/Permission.jsx (hunk 2 above)
Comment (vdnj) [reply src/containers/Permission.jsx:36]: This text will be removed, it is just a placeholder at the moment.

File: src/containers/Permission.jsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+          </Typography>
+          <Typography variant="h2">Permission: {permissionName}</Typography>
```
[the remark this answers is not in the document]
Comment (vdnj) [reply src/containers/Permission.jsx:38]: see previous comment

File: src/containers/Permission.jsx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+          <List>
+            {queryResult.data[0].attributes.permissions[permissionName]
+              .verbs ? (
+              queryResult.data[0].attributes.permissions[
+                permissionName
+              ].verbs.map((el, i) => <ListItemText key={i}>{el}</ListItemText>)
+            ) : (
+              <ListItemText>ALL</ListItemText>
```
[the remark this answers is not in the document]
Comment (vdnj) [reply src/containers/Permission.jsx:46]: see previous comment

**Review by trollepierre — APPROVED** [review]

**Pull request merged**